addons/mail/tests/common.py

```diff
@@ -913,7 +914,12 @@ def assertBus(self, channels, message_items=None):
                 yield
         finally:
             found_bus_notifs = self.assertBusNotifications(channels, message_items=message_items)
-            self.assertEqual(self._new_bus_notifs, found_bus_notifs)
+            self.assertEqual(
+                self._new_bus_notifs,
+                found_bus_notifs,
+                f"\n{self._new_bus_notifs.mapped(lambda bus: (bus.channel, json.loads(bus.message).get('type')))}"
```

Comment:
`nitpick` Is there a way to get rid of the database name, it is annoying seeing the redundant stuff. Or put it in the second place rather than the first.

Reply:
We could post-process the bus message assuming they have a known shape to remove what we want, but that's how they are encoded in the first place so I'm not sure if it's worth it.

As long as we can see the diff it's the most important.